extensions/Worker.Extensions.Timer/src/TimerExtensionStartup.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.Functions.Worker.Core;
+using Microsoft.Azure.Functions.Worker.Extensions.Timer;
+using Microsoft.Azure.Functions.Worker.Extensions.Timer.Converters;
+using Microsoft.Extensions.DependencyInjection;
+
+[assembly: WorkerExtensionStartup(typeof(TimerExtensionStartup))]
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.Timer
+{
+    /// <summary>
+    /// Startup for Microsoft.Azure.Functions.Worker.Extensions.Timer extension.
+    /// </summary>
+    public sealed class TimerExtensionStartup : WorkerExtensionStartup
+    {
+        /// <inheritdoc/>
+        public override void Configure(IFunctionsWorkerApplicationBuilder applicationBuilder)
+        {
+            if (applicationBuilder == null)
+            {
+                throw new ArgumentNullException(nameof(applicationBuilder));
+            }
+
+            applicationBuilder.Services.Configure<WorkerOptions>((workerOption) =>
+            {
+                workerOption.InputConverters.RegisterAt<TimerPocoConverter>(0);
```

Comment:
Can we instead specify the converter on the `TimerTriggerAttribute`? We can also have this be a nested class of that attribute so it is easy to identify that this converter applies to only that type.

``` CSHarp
[InputConverter(typeof(Converter))]
[ConverterFallbackBehavior(ConverterFallbackBehavior.Disallow)]
public sealed class TimerTriggerAttribute : InputBindingAttribute
{
    internal class Converter : IInputConverter
    {
    }
}
```

Nesting the class is just a suggestion - if you want to leave it separate that is fine. But we should use the attribute - avoids adding global converters.

Reply:
I considered that. But that would be a breaking change since `TimerTriggerAttribute` is a public type and if any customer has written code to check the attributes on this type and did anything based on that, it will break.

Reply:
I wouldn't worry about that as a breaking change. Attributes lay in a grey area for breaking changes - the attribute has to be documented as a feature we expect the customer to interact with directly. This attribute is not advertised/documented like that. Any customer usage of it will fall under unsupported reflection, and not be considered a breaking change.

Reply:
Thoughts on this @fabiocav @brettsam ?

Reply:
Discussed during standup. Team agrees that this can be considered a non-breaking change. I will go ahead and make the changes.

Reply:
Pushed an iteration reflecting this change.